cockatrice/src/settingscache.cpp

```diff
@@ -37,9 +37,12 @@ SettingsCache::SettingsCache()
     picDownloadHq = settings->value("personal/picturedownloadhq", true).toBool();
 
     picUrl = settings->value("personal/picUrl", PIC_URL_DEFAULT).toString();
-    picUrlHq = settings->value("personal/picUrlHq", PIC_URL_HQ_DEFAULT).toString();
+    //picUrlHq = settings->value("personal/picUrlHq", PIC_URL_HQ_DEFAULT).toString();
     picUrlFallback = settings->value("personal/picUrlFallback", PIC_URL_FALLBACK).toString();
     picUrlHqFallback = settings->value("personal/picUrlHqFallback", PIC_URL_HQ_FALLBACK).toString();
+    
+    picUrlHq = settings->value("personal/highQualityURL", PIC_URL_HQ_DEFAULT).toString();
+    
```

Comment:
You renamed the url option for the primary high quality card. But tree other options exists (low/high quality, primary/fallback urls). We'd better rename them all to be consistent.